package.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
I think that we should change the `common` section and the `repository.url` as-well

Reply:
I'm not sure what is expected to be put in there, so I'll probably need some help with this.

Reply:
We need to remove the `common` section only